lib/stamp/emitters/am_pm.rb

```diff
@@ -1,22 +1,22 @@
 module Stamp
   module Emitters
     class AmPm
-      include Modifiable
-
-      AM = 'am'
-      PM = 'pm'
-
-      def initialize(&block)
-        @modifier = block
+      # @param [Hash] options
+      # @option options [Boolean] :upcase (default: false) uppercase meridian
```

Comment:
I fixed the spelling. renaming am pm is all on you.

But thanks for the grammar correction. very cool